README.md

```diff
@@ -128,132 +128,6 @@ Then perform the following commands on the root folder:
 - `terraform destroy` to destroy the built infrastructure
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-## Inputs
```

Comment:
Thanks for the pull request @TheKangaroo.

The Readme(s) appear to have missing content.  Can you try running `make docker_generate_docs`?

Reply:
sorry, should be fixed now